src/sage/categories/schemes.py

```diff
@@ -1,14 +1,18 @@
 r"""
 Schemes
 """
+
 # ****************************************************************************
-#  Copyright (C) 2005      David Kohel <kohel@maths.usyd.edu>
+#       Copyright (C) 2013 David Kohel <kohel@maths.usyd.edu>
 #                          William Stein <wstein@math.ucsd.edu>
```

Comment:
Is this change of the copyright year correct?

Reply:
Thanks for spotting this.

Fixed.